superdesk/notification.py

```diff
@@ -58,7 +58,7 @@ def _notify(**kwargs):
     def send_message():
         yield from app.notification_client.send(message)
 
-    loop = asyncio.get_event_loop()
+    loop = asyncio.new_event_loop()
```

Comment:
Without this change at least locally the notification is not working at all.
With this change the notification don't work if app.py is started after ws.py.

I added here and to https://github.com/superdesk/superdesk-core/blob/fix/notification-loop-multi-thread/superdesk/notification.py#L43 the following code:
```
        try:
            loop = asyncio.get_event_loop()
        except:
            loop = asyncio.new_event_loop()
```

and at least locally it is working well.